### ISSUE TITLE
rio ps port listing is hard to read

### ISSUE THREAD

**Issue by apollo13** [rancher/rio#743]
**Describe the bug**
`rio ps` includes a listing of ports which becomes hard to read when the ports contains endpoint names (or whatever the later is called). Consider this part of a `Riofile`:

```
    ports:
    - 80/http,webtest
    - 8888:80/tcp,admin,internal=false
```
which results in:
```
PORTS
80,webtest,8888:80/tcp,admin
```
on display. Looking through `kubectl get -A service`, k8s uses comma as separator between ports, so it would be nice to stay consistent here and keep the comma as port separator. One option would be to use `@` like this:
```
PORTS
webtest@80,admin@8888:80/tcp
```
or putting the name (and possible options like `internal` into `[]`):
```
PORTS
80[webtest],8888:80/tcp[admin,internal=false]
```

If you have a preference I can give a PR a shot.

**Comment by StrongMonkey** [comment]
@apollo13 https://github.com/rancher/rio/blob/master/cli/pkg/tables/service.go#L58 Check this out

**Comment by apollo13** [comment]
@StrongMonkey yeah, I already found this code. But which of the two formats I presented would you prefer? FWIW I'd keep comma as separator for consistency with the rest of k8s.

**Comment by StrongMonkey** [comment]
Colon sounds good to me, but @vincent99 may have an opinion.
